### PR TITLE
Create nature-sciences-societes.csl

### PR DESCRIPTION
**Pull request by POBrien333**
https://forums.zotero.org/discussion/70728/style-request-natures-sciences-societes#latest

### CODE REVIEW

**Comment by adam3smith** [comment]
Thanks!

File: nature-sciences-societes.csl [hunk 1]
```diff
@@ -0,0 +1,302 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Natures Sciences Sociétés</title>
+    <title-short>NSS</title-short>
+    <id>http://www.zotero.org/styles/nature-sciences-societes</id>
```
Comment (rmzelle) [inline nature-sciences-societes.csl]: Style ID is missing an "s" ("nature" should be "natures"). Will fix this up.

Comment (rmzelle) [reply nature-sciences-societes.csl]: Fixed in https://github.com/citation-style-language/styles/pull/3420.

**Comment by rmzelle** [comment]
@POBrien333, thanks! By the way, I assume you just checked this style for English for this bilingual journal? Or does it work for French as well?

**Pull request merged**